tests/test_catalogue.py

```diff
@@ -82,6 +84,44 @@ def test_catalogue_format():
                 assert flux_err != 0.
 
 
+def test_convert_antf_ref():
+    ref_dict = get_antf_references()
+
+    # Get ref codes for all pulsar fluxes
+    query = psrqpy.QueryATNF(version=ATNF_VER).pandas
+    flux_queries = []
+    for table_param in query.keys():
+        if re.match("S\d*\d$", table_param) or re.match("S\d*G$", table_param):
```

Comment:
Pytest returns a warning for this line: "invalid escape sequence '\d'". Is this okay?

Reply:
I've added an r in front of those strings to declare that it is a re pattern, which removed the warning.